tools/bazel/BUILD

```diff
@@ -0,0 +1,7 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
```

Comment:
For awareness, this is added to allow users to generate documentation through Bazel's [stardoc](https://github.com/bazelbuild/stardoc) tool. If a downstream user writes a `*.bzl` file that loads any of the `*.bzl` files here, they will need a [bzl_library](https://github.com/bazelbuild/bazel-skylib/blob/master/bzl_library.bzl) containing the files here available to their workspace in order to generate documentation.